components/playlist/playlist_service.h

```diff
@@ -149,6 +151,26 @@ class PlaylistService : public KeyedService,
       content::WebContents* web_contents,
       blink::web_pref::WebPreferences* web_prefs);
 
+  // playlist::mojom::Service:
```

Comment:
the mojo interface should be the only methods we use, we should not be duplicating the existing methods. The non-mojo versions should be removed